File: square.js
function sqrt(number) {
  var counter = 2;
  while (counter < number) {
    if (counter * counter == number) {
      return counter;
    }
    counter += 1;
  }
  return 0;
};

console.log(`The root of 25 is ${sqrt(25)}`);
console.log(`The root of 100 is ${sqrt(100)}`);
console.log(`The root of 1000000 is ${sqrt(1000000)}`);
console.log(`The root of 1000000000000000000 is ${sqrt(1000000000000000000)}`);

File: square.go
package main

import "fmt"

func sqrt(number int) int {
  counter := 2;
	for counter < number {
		if ((counter * counter) == number) {
		  return counter;
		}
		counter++
	}
  return 0;
}

func main() {
	fmt.Printf("The root of 25 is  %v\n", sqrt(25))
	fmt.Printf("The root of 100 is  %v\n", sqrt(100))
	fmt.Printf("The root of 1000000 is  %v\n", sqrt(1000000))
	fmt.Printf("The root of 1000000000000000000 is  %v\n", sqrt(1000000000000000000))
}
